socs/agents/ibootbar/agent.py

```diff
@@ -198,6 +201,7 @@ def __init__(self, agent, address, port=161, version=2, lock_outlet=None):
         self.snmp = SNMPTwister(address, port)
 
         self.lastGet = 0
+        self.sample_rate = 60
```

Comment:
Meaning of "period" vs "rate" is important for clarity.  Please change to sample_period.